Dockerfile

```diff
@@ -1,12 +1,12 @@
-# Use the latest Debian unstable slim image as the base
-# TODO: Move back to stable when nightlies are fixed
-FROM debian:unstable-slim
+# Use the latest Ubuntu image as the base
+FROM ubuntu:rolling
 
 # Make sure that all packages are up to date then
-# install the base Debian packages that we need for
-# building the kernel and QEMU
+# install the base Ubuntu packages that we need for
+# building the kernel
 RUN apt-get update -qq && \
     apt-get upgrade -y && \
+    ln -snf /usr/share/zoneinfo/UTC /etc/localtime && echo "UTC" > /etc/timezone && \
```

Comment:
why is this required?

Reply:
https://travis-ci.com/nathanchance/dockerimage/builds/92513642

There doesn't appear to be a default timezone.

Reply:
haha interesting, thanks for the background